app/src/main/java/org/schabi/newpipe/error/ReCaptchaActivity.java

```diff
@@ -108,11 +107,8 @@ public void onPageFinished(final WebView view, final String url) {
         recaptchaBinding.reCaptchaWebView.clearCache(true);
         recaptchaBinding.reCaptchaWebView.clearHistory();
         final CookieManager cookieManager = CookieManager.getInstance();
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
-            cookieManager.removeAllCookies(value -> { });
-        } else {
-            cookieManager.removeAllCookie();
-        }
+        cookieManager.removeAllCookies(value -> {
+        });
```

Comment:
I don't believe this callback actually does anything. I think it can just be replaced with `cookieManager.removeAllCookies(null)`.

Reply:
done